spec/controllers/application_controller_spec.rb

```diff
@@ -122,4 +122,12 @@ def index
       end
     end
   end
+
+  describe '#analytics' do
+    it 'uses the Analytics class by default' do
+      expect(Analytics).to receive(:new).with(controller.current_user, request)
```

Comment:
I don't think they are a detail. It's similar to when we test that an SMS background job receives the correct phone number as an argument. I will update the test title.